app/src/main/java/org/piwigo/io/repository/PreferencesRepository.java

```diff
@@ -49,4 +55,13 @@ private void set(String key, String value) {
         editor.putString(key, value);
         editor.apply();
     }
+
+    public String getString(String key, String value) {
```

Comment:
good that you integrated the defaults in this class, but so far the user still have to pass it into getXXX. Why not remove the value parameter completely and handle the default inside PreferenceRepository? I think that would be even more clean than as a user first asking the PreferenceRepository for the default and then pass it back as "value" into it to eventually get it back again...

Reply:
ok